planet/src/main/webapp/index.html

```diff
@@ -41,10 +50,45 @@ <h1>Get ready to
             </div>
         </div>
     </div>
-    <div class="section white middle" id="next">
-        <div class="row container">
-            <p>Insert text here. To be determined.</p>
+    <div id="next">              
+    <a id="up" href=# class="waves-effect waves-light btn-floating btn red accent-2"><i class="material-icons">expand_less</i></a>        
+        <div class="row card-container center">
+            <div class="column center">
+                <div class="card">
+                    <h3>Step 1:</h3>
+                    <h3>Create a profile</h3>
+                    <hr/>
+                    <p>Sign up or log in to get started. Create a profile and connect with other travellers. View the travellers visiting the same places as you.</p>
+                    <a class="go waves-effect waves-light btn red accent-2">GO</a>
+                </div>                
+            </div>
+            <div class="column center">
+                <div class="card">
+                    <h3>Step 2:</h3>
+                    <h3>Search Destinations</h3>
+                    <hr/>
+                    <p>Navigate to the Planet home page and start searching and saving destinations. View your searched locations on our map and browse through a list
+                        with details about each place.</p>
+                    <a href="places.html" class="go waves-effect waves-light btn red accent-2">GO</a>
```

Comment:
Consider adding different text to the different buttons. Maybe something like "Get Started", "Start Searching", and "Plan"?

Reply:
(Feel free to choose better wording)

Reply:
Sounds good! will do that